softcut-lib/include/softcut/Assert.h

```diff
@@ -0,0 +1,6 @@
+#ifndef Softcut_ASSERT_H
+#define Softcut_ASSERT_H
+
+#define ASSERT_MSG(expr, msg) assert((expr)&&(msg))
```

Comment:
this isn't a very nice way to get the error message - it will only be shown kinda by accident as part of the stacktrace, instead of going to `stderr`